src/kademlia/dht_storage.cpp

```diff
@@ -119,9 +119,18 @@ namespace
 		std::memcpy(item.value.get(), buf.data(), size);
 	}
 
+	system_clock::time_point clock_now()
+	{
+		static auto const stime = system_clock::now();
+		static auto const ctime = aux::time_now();
+
+		auto const secs = total_seconds(aux::time_now() - ctime);
+		return stime + seconds(secs);
```

Comment:
ok ok...I think I can use the old structure, and save a `timespan` as a reference when saving/loading